app/views/restaurants/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="w-25 container">
+  <%= form_with model: @restaurant do |form| %>
+    <%if @restaurant.errors.any? %>
+      <div class="alert alert-danger">
+        <% @restaurant.errors.full_messages.each do |message| %>
+          <div><%= message %></div>
+        <% end %>
+      </div>
```

Comment:
Extract the error rendering logic in a shared partial. I think we have same logic for users views as well.